src/bentoml/_internal/configuration/v1/default_configuration.yaml

```diff
@@ -32,6 +56,7 @@ api_server:
       access_control_allow_credentials: ~
       access_control_allow_methods: ~
       access_control_allow_headers: ~
+      access_control_allow_origin_regex: ~
```

Comment:
Shall we add this to `access_control_options` as well?

Reply:
@aarnphm

Reply:
sure